modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators/ExtensionMethods.cs

```diff
@@ -223,6 +233,13 @@ private static void FullQualifiedSyntax(SyntaxNode node, SemanticModel sm, Strin
             }
         }
 
+        private static void ParenEnclosedFullQualifiedSyntax(SyntaxNode node, SemanticModel sm, StringBuilder sb, bool isFirstNode)
```

Comment:
This method is only used in `FullQualifiedSyntax` and I don't expect we'd ever use it somewhere else, so it can be a local method.